src/Compilers/CSharp/Portable/Symbols/SpecialTypeExtensions.cs

```diff
@@ -63,6 +63,29 @@ public static bool CanBeConst(this SpecialType specialType)
             }
         }
 
+        public static bool IsValueType(this SpecialType specialType)
```

Comment:
Is this the same as a check for if the type is blittable? I could have sworn we already have a helper for this...